### DISCUSSION TITLE
Can we download a translated ebook (not dual language but only in translated language)

### DISCUSSION THREAD

**Discussion by william-ng-ln** [immersive-translate/immersive-translate#1024] [Ideas]
### 新功能 | New Feature

I think it's better to have an option to download a translated ebook (not dual language but only in translated language)

### 为什么您想要添加这些功能 | Why you want to add these features

Sometimes I think it is easy to read, especially a very long book.

**Comment by theowenyoung** [comment]
yes, you can, just click [More] [Show translation only], then translate the ebook.
